File: tugas 823/tugas nilai/nilai.py
nilai = 90

match (nilai):
    case n if n >= 90:
        print("Nilaimu A")
    case n if n >= 80:
        print("Nilaimu B")
    case n if n >= 70:
        print("Nilaimu C")
    case n if n >= 60:
        print("Nilaimu D")
    case n if n < 60:
        print("Nilaimu E")

File: tugas 823/tugas nilai/nilai.js
let nilai = 90;

switch (true) {
    case (nilai >= 90):
        console.log("Nilaimu A");
        break;
    case (nilai >= 80):
        console.log("Nilaimu B");
        break;
    case (nilai >= 70):
        console.log("Nilaimu C");
        break;
    if (nilai >= 60) {
        console.log("Nilaimu D");
    }
    default:
        console.log("Nilaimu D");
        break;
}
